style.css

```diff
@@ -0,0 +1,7 @@
+/* Intended for h1 elements within the myh1 class */
+.myh1 {
+  margin: 25px;
+  background-color: rgb(240, 240, 240);
+  color: rgb(38, 95, 62);
+  font-family: arial, sans-serif;
+}
```

Comment:
[OPTIONAL] Kindly make this information as a hyperlink, you can do that by surrounding username with [] then put the link between ().
```
[username](url)
```

Reply:
Thank you for your review! Still, I don't understand the Optional, where should I put the link? (inside the style.css?)

Reply:
No, for the README links that contain your information like GitHub, Twitter and LinkedIn.

Reply:
This part
![image](https://user-images.githubusercontent.com/24703171/158495469-c194affc-1e37-4d47-ac0c-9ce23bb3861d.png)